### ISSUE TITLE
Wrong Mouse pointer during drag-and-drop

### ISSUE THREAD

**Issue by probonopd** [lxqt/pcmanfm-qt#1476]
##### Expected Behavior

Mouse pointer (cursor shape) should stay an arrow during drag-and-drop in the file manager

![image](https://user-images.githubusercontent.com/2480569/140326247-a5b272cb-e0ec-47ec-9050-7792663ad28f.png)

##### Current Behavior

Mouse pointer (cursor shape) becomes an __X__

![image](https://user-images.githubusercontent.com/2480569/140325771-e8afcb97-c271-445a-ad85-8c038570c9b8.png)

during the drag operation, suggesting that the action is not possible. But in fact it is possible.

##### Possible Solution

Likely something to do with https://doc.qt.io/qt-5/qdrag.html#setDragCursor and/or https://doc.qt.io/qt-5/qcursor.html

##### Steps to Reproduce (for bugs)
1. Open pcmanfm-qt
2. Open a folder
3. Move an item within the window
4. Look at the shape of the mouse pointer (cursor)

##### Context

Possibly related to https://github.com/lxqt/pcmanfm-qt/issues/1086

##### System Information

* PcManFm-qt version: 0.17.0
* Distribution & Version: FreeBSD 12.2
* Qt Version: 5.15.2
* liblxqt Version: 0.17.0_1

**Comment by tsujan** [comment]
If it happened, we'd notice it immediately. Not reproducible:

![dnd](https://user-images.githubusercontent.com/981076/140400265-980937ae-7cc0-4d2b-ba87-eb6e2c6547be.png)

> Possibly related to #1086

That's just about info; it has no code.

You need to find the necessary conditions for reproducing it. If reproducible, I'll surely re-open this page.

**Comment by probonopd** [comment]
Is there a dependency on a certain cursor theme? (I am using the stock Xorg one.)

**Comment by tsujan** [comment]
> Is there a dependency on a certain cursor theme?

None at all. Cursors are handled by Qt.

**Comment by tsujan** [comment]
There was no need to a screen record; I definitely believed that you see what you described ;)

**Comment by iamthesenate1** [comment]
Something that I must mention is that at some point, on a debian instalation I had the same problem. The issue was happening only when lxqt was started with startx. Starting with sddm "fixed" it. The problem was happening when I dragged anything anywhere so I don't think that lxqt is at fault

**Comment by probonopd** [comment]
fwiw, I am not using sddm but [SLiM](https://en.wikipedia.org/wiki/SLiM).

**Comment by tsujan** [comment]
@probonopd 
Do you see it everywhere or only in pcmanfm-qt?

**Comment by stefonarch** [comment]
It looks like the x is where usually the + sign appears when dragging. Did you try other mouse themes?

**Comment by iamthesenate1** [comment]
> It looks like the x is where usually the + sign appears beside the cursor when dragging. Did you try other mouse themes?

This would make sense, at least in my case, as startx loaded the adwaita cursor theme while sddm loaded breeze.

**Comment by probonopd** [comment]
This is why I was asking about "cursor theme". Will try another one.

**Comment by probonopd** [comment]
Confirmed. When I set the mouse theme to "Breeze by the KDE VDG", then it works.

But I think the "+" is actually not what should be shown, since only after the user lets go of the mouse the user decides whether to move, symlink, or actually copy the item. So at the point when the drag begins there is no way of knowing yet whether the "+" or a symlink symbol or nothing should be shown. Unless the user presses a modifier key.
